Resources/include/StringManipulationFunctions.hpp

```diff
@@ -12,6 +12,15 @@
 #include <vector>
 
 namespace StringManipulation {
+    ///Converts full string to lower case. 
+    /// @param [in] s : String to make lowercase
+    /// @return the lowercase string
+    inline std::string StringLower(std::string s){
+        std::transform(s.begin(),s.end(),s.begin(),::tolower);
```

Comment:
Maybe add std to the :: here? Not really necessary, just nitpicky

Reply:
Can't use std::tolower without passing in a lambda function, and to support older compilers don't want to make that possibly breaking change